<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDiffRateItemParaEditWidget</class>
 <widget class="QWidget" name="QDiffRateItemParaEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>895</width>
    <height>355</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,4,2,4,1">
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="m_labRate_FaultType">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>故障相别:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="m_cmbRate_FaultType">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>A</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ABC</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>AB</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>BC</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CA</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="m_labRate_BeginVal">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>搜索上限(%):</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_txtRate_BeginVal">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="m_labRate_EndVal">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>搜索下限(%):</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_txtRate_EndVal">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="m_labRate_Step">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>测试精度():</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_txtRate_Step">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="m_labRate_Freq">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>频率(Hz):</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_txtRate_Freq">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="m_labRate_Ir">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>制动电流Ir():</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_txtRate_Ir">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="m_labRate_IdSetting">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>差动理论值Id():</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_txtRate_IdSetting">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="m_labRate_FaultTime">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>最大故障时间(s):</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_txtRate_FaultTime">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="m_labRate_AbsErr">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>绝对误差():</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSettingLineEdit" name="m_txtRate_AbsErr">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="m_labRate_RelErr">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>相对误差(%):</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSettingLineEdit" name="m_txtRate_RelErr">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSettingLineEdit</class>
   <extends>QLineEdit</extends>
   <header>../../UI/Controls/SettingCtrls/qsettinglineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
